report/assets/styles.css

```diff
@@ -1168,16 +1168,14 @@
 }
 
 .lh-gauge-base {
-    opacity: 0.1;
-    stroke: var(--circle-background);
```

Comment:
these intentionally removed? having trouble seeing how these are related

Reply:
these variables do not exist. not related, just found during debugging.